### ISSUE TITLE
java.lang.NoClassDefFoundError: io/netty/internal/tcnative/SSLPrivateKeyMethod

### ISSUE THREAD

**Issue by vspiewak** [spotify/scio#2933]
Hi !

I created a new SCIO job using [scio.g8](https://github.com/spotify/scio.g8) template.
Added scio-bigquery and play-json dependencies.

`sbt clean package` give me this warning 
`[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.`

When I try to run the job, I get this error :
```
[error] (run-main-0) java.lang.NoClassDefFoundError: io/netty/internal/tcnative/SSLPrivateKeyMethod
[error] java.lang.NoClassDefFoundError: io/netty/internal/tcnative/SSLPrivateKeyMethod
[error]         at io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:824)
[error]         at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:462)
[error] 	at com.spotify.scio.pubsub.PubSubAdmin$GrpcClient$.newChannel(PubSubAdmin.scala:43)
[error] 	at com.spotify.scio.pubsub.PubSubAdmin$GrpcClient$.publisher(PubSubAdmin.scala:58)
[error] 	at com.spotify.scio.pubsub.PubSubAdmin$.ensureTopic(PubSubAdmin.scala:76)
[error] 	at [redacted].BackupRaw$.main(BackupRaw.scala:80)
[error] 	at [redacted].BackupRaw.main(BackupRaw.scala)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] 	at java.lang.reflect.Method.invoke(Method.java:498)
[error] Caused by: java.lang.ClassNotFoundException: io.netty.internal.tcnative.SSLPrivateKeyMethod
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
[error] stack trace is suppressed; run last Compile / bgRunMain for the full output
[error] Nonzero exit code: 1
[error] (Compile / runMain) Nonzero exit code: 1
[error] Total time: 3 s, completed Apr 30, 2020 5:45:44 PM
```

How can I solve this ?

Here is my environment
```
openjdk version "1.8.0_242"
OpenJDK Runtime Environment (build 1.8.0_242-b08)
OpenJDK 64-Bit Server VM (build 25.242-b08, mixed mode)
```
```
sbt version in this project: 1.3.10
sbt script version: 1.3.9
```

**Comment by vspiewak** [comment]
Here is the ouput of `sbt evicted`
[evicted.log](https://github.com/spotify/scio/files/4559424/evicted.log)

**Comment by regadas** [comment]
@vspiewak Thanks for the report! This might be related #2915.

**Comment by regadas** [comment]
@vspiewak can I push you to give `0.9.1-SNAPSHOT` a try, just to see if you face the same issue? 

```scala
resolvers += Resolver.sonatypeRepo("snapshots")
```

Also what runner are you using?

**Comment by vspiewak** [comment]
I'm still facing the same issue using `0.9.1-SNAPSHOT` and dataflow runner :disappointed:

**Comment by vspiewak** [comment]
I fixed the issue related to `PubSubAdmin.ensureTopic` using :

- Scala `2.11.12`
- Beam `2.19.0`
- Scio `0.8.4`
- `dependencyOverrides += "io.opencensus" % "opencensus-api" % "0.21.0"`

I saw a lot of dependencyOverrides in Scio build. 
I guess the issue in `0.9.0` can come from others dependencies...

**Comment by vspiewak** [comment]
**For info**: ATM, creating a new SCIO project using scio.g8 template and adding `PubSubAdmin.ensureTopic` call in `WordCount.scala` will give you the error on my first comment.

Using `0.9.1-SNAPSHOT` didn't changed

**Comment by regadas** [comment]
@vspiewak Thanks for the extra info! I'll look into it!

**Comment by regadas** [comment]
@vspiewak got some time to have a look at this and if you want to get going you will need to add: 

```scala
ThisBuild / dependencyOverrides ++= Seq(
  "io.netty" % "netty-tcnative-boringssl-static" % "2.0.30.Final",
  "com.google.guava" % "guava" % "27.0.1-jre"
)
```

Not sure if yet if we can provide a fix `scio` side ...

**Comment by vspiewak** [comment]
Dumb question, this will not be fixed if you add it here [build.sbt](https://github.com/spotify/scio/blob/bbf73265c2df58d59b2507673409df487932324f/build.sbt#L1277) ?

I saw the `netty-tcnative-boringssl-static` but was a bit lost with all the `evicted` output !

**Comment by regadas** [comment]
@vspiewak that would be great but that doesn't get reflected in `pom.xml` for artifacts published maven style.

If this was ivy then it would work.

**Comment by regadas** [comment]
@vspiewak Ok published a new `0.9.1-SNAPSHOT` that should resolve this issue! can you give it a try?
